src/ralph/data_importer/resources.py

```diff
@@ -594,56 +592,59 @@ class Meta:
 
 class BaseObjectsSupportRichResource(RalphModelResource):
     price_per_object = fields.Field(
-        column_name='support__price_per_object',
+        column_name="support__price_per_object",
     )
 
     class Meta:
         model = BaseObjectsSupport
         fields = (
-            'support__contract_id',
-            'support__support_type',
-            'support__serial_no',
-            'support__name',
-            'support__price',
-            'price_per_object',
-            'support__date_from',
-            'support__date_to',
-            'support__description',
-            'support__invoice_no',
-            'support__invoice_date',
-            'support__property_of',
-            'support__budget_info',
-            'baseobject__asset__hostname',
-            'baseobject__asset__barcode',
-            'baseobject__asset__sn',
-            'baseobject__asset__model',
-            'baseobject__service_env',
-            'baseobject__configuration_path',
+            "support__contract_id",
+            "support__support_type",
+            "support__serial_no",
+            "support__name",
+            "support__price",
+            "price_per_object",
+            "support__date_from",
+            "support__date_to",
+            "support__description",
+            "support__invoice_no",
+            "support__invoice_date",
+            "support__property_of",
+            "support__budget_info",
+            "baseobject__asset__hostname",
+            "baseobject__asset__barcode",
+            "baseobject__asset__sn",
+            "baseobject__asset__model",
+            "baseobject__service_env",
+            "baseobject__configuration_path",
         )
 
-    def get_queryset(self):
-        return super().get_queryset().annotate(
-            objects_count=Count('support__baseobjectssupport')
-        )
+    # def get_queryset(self):
+    #     return (
+    #         super()
+    #         .get_queryset()
+    #         .annotate(objects_count=Count("support__baseobjectssupport"))
+    #     )
```

Comment:
Decide what to do with it.
Otherwise python 3.8 complains with garbage
`RuntimeError: __class__ not set defining 'BaseObjectsSupportRichResource' as <class 'ralph.data_importer.resources.BaseObjectsSupportRichResource'>. Was __classcell__ propagated to type.__new__?`

or when fixed in ImportForeignKeyMeta:

`TypeError: __class__ set to <class 'ralph.data_importer.resources.BaseObjectsSupportRichResource'> defining 'BaseObjectsSupportRichResource' as <class 'ralph.data_importer.resources.BaseObjectsSupportRichResource'>`